### ISSUE TITLE
Unable to input values with spaces in the tag-input-form

### ISSUE THREAD

**Issue by tonysparks** [Gbuomprisco/ng2-tag-input#143]
I'm unable to input values with spaces, the KeyEvent with keyCode of 32 is eaten via the following code:

[actions.ts](https://github.com/Gbuomprisco/ng2-material-dropdown/blob/master/src/modules/components/menu/actions.ts)
```typescript
export function arrowKeysHandler(event): void {
    if ([32, 37, 38, 39, 40].indexOf(event.keyCode) > -1) {
        event.preventDefault();
    }
}
```

![image](https://cloud.githubusercontent.com/assets/1552968/21687595/827e0352-d32f-11e6-8008-c5d626c93575.png)

**Comment by Gbuomprisco** [comment]
Hi @tonysparks,

Thanks for reporting it. I opened a PR, I'll merge as soon as I verified it. This also fixes https://github.com/Gbuomprisco/ng2-tag-input/issues/142

https://github.com/Gbuomprisco/ng2-tag-input/pull/144

**Comment by Gbuomprisco** [comment]
This should be fixed with version 0.7.2
